db/db.go

```diff
@@ -0,0 +1,71 @@
+package db
+
+import (
+	"fmt"
+	"time"
+
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+)
+
+func GetDB() *gorm.DB {
+	db, err := gorm.Open(sqlite.Open("test.db"), &gorm.Config{})
+	if err != nil {
+		panic("failed to connect database")
```

Comment:
Do we use panic or log.Fatal? To have some information about the time the failure occurred maybe? We should add to the spec the way we handle errors.

Reply:
yeah, a will change it to `log.Fatal`. And I agree that we should do kind of coding guidelines spec

Reply:
also I want to notice that db setup happening once at startup

Reply:
>  change it to log.Fatal

done